feat: import and handle dataviz tokens

BalbinaK opened the pull request:
## Description

Fixes [WARP-644](https://nmp-jira.atlassian.net/browse/WARP-644)

This PR adds logic to import DataViz tokens, transform them and generate assets for web, iOS and Android using StyleDictionary. The iOS and Android part has not been looked at, but the colors are generated and zipped.
The build script was refactored to cover both DataViz and Warp tokens and most logic was stored in `buildUtils.js`.

## Testing
Run `pnpm build:dataviz` (you will need the Figma token) and check the `/output` folder for generated assets.

BalbinaK committed:
Import and process dataviz tokens (colors)

BalbinaK committed:
process semantic dataviz color tokens

BalbinaK committed:
use StyleDictionary to generate css

BalbinaK committed:
Update dagaVizFigma.json

BalbinaK committed:
Refactor processColors functions

BalbinaK committed:
move common build logic to buildUtils.js

felicia-haggqvist reviewed (approved):
Really nice work! 🥳 🎉

BalbinaK committed:
Fix typo in Figma token message

- Corrected viariables to variables in success message

imprashast reviewed (approved):
Good one. Nice refactors 👍

BalbinaK commented:
Before I merge I think I need to revisit the build scripts so I can add the dataviz tokens fetching and release to the github workflows.

BalbinaK committed:
fix: filter out unsupported brands (#14)

BalbinaK committed:
Merge remote-tracking branch 'origin/main' into feat/handle-dataviz-tokens

BalbinaK committed:
remove charttext, chartline and black token

BalbinaK committed:
Update figma.json

BalbinaK committed:
Remove warpgray and chartbackground token

BalbinaK committed:
Format datavizFigma.json

BalbinaK committed:
make the JSON files more readable by adding an indentation argument to JSON.stringify

BalbinaK committed:
Add workflows for fetching Figma tokens and update structure

- Create a new GitHub Action to fetch dataviz tokens from Figma.
- Rename existing workflow to focus on fetching warp tokens.
- Update import scripts for dataviz and warp tokens.
- Only build assets in the release workflow

BalbinaK commented:
@imprashast before I merge, I'd like you to have a look at the last commit: [Add workflows for fetching Figma tokens and update structure](https://github.com/warp-ds/tokens/pull/13/commits/0ede0ff49ff4750a4652135477d60faf69405d50) 
[0ede0ff](https://github.com/warp-ds/tokens/pull/13/commits/0ede0ff49ff4750a4652135477d60faf69405d50)
Changes included there:
- Create a new GitHub Action to fetch dataviz tokens from Figma.
- Rename existing workflow to focus on fetching warp tokens.
- Update import scripts for dataviz and warp tokens.
- Only build assets in the release workflow
